app/grandchallenge/workstations/static/workstations/js/session-region.js

```diff
@@ -0,0 +1,46 @@
+function setModalLoadingMessage(msg) {
+    document.getElementById("sessionStateMsg").innerHTML = msg;
```

Comment:
It is generally better to avoid `innerHTML` to prevent XSS attacks unless you explicitly want to add child elements by text. An XSS attack is not possible here, but still, it is good practice to use `textContent` or `innnerText` here since you are only setting text.